lib/mix/tasks/popcorn.build_runtime.ex

```diff
@@ -59,6 +59,10 @@ defmodule Mix.Tasks.Popcorn.BuildRuntime do
         build_dir = Path.join(runtime_source, "src/platforms/emscripten/build")
         File.mkdir_p!(build_dir)
 
+        if System.find_executable("emcmake") == nil do
+          raise "emcmake command not found, please make sure you have emscripten installed"
```

Comment:
If we check that, we can also check for `cmake`, `make`:
```elixir
ensure_executables!(["emcmake", "cmake", "make"])

defp ensure_executables!(execs) do
  has_executable? = fn name -> name |> System.find_executable() |> is_binary() end
  for exe <- execs do
    if not has_executable?.(exe) do
      raise "#{exe} command not found, please make sure you have emscripten and C tooling installed"
    end
  end
end
```

Reply:
> I think having emcmake implies having the rest

Not necessarily, on my Manjaro, the `emscripten` package does not depend on either cmake or make